sdk/keyvault/keyvault-keys/test/utils/testAuthentication.ts

```diff
@@ -23,8 +23,6 @@ export async function authenticate(that: Context, version?: string): Promise<any
       AZURE_MANAGEDHSM_URI: "https://azure_managedhsm.managedhsm.azure.net"
     },
     customizationsOnRecordings: [
-      (recording: any): any =>
```

Comment:
One less thing for me as a package owner to worry about 😄 

Other than just knowledge / experience in our repo is there a place where things that get sanitized by default are defined? I'm thinking about future me that will likely forget that `access_token` is sanitized by default

Reply:
Good point.. not present as of now.
We can add a section in the readme.

Reply:
#14416